index.bs

```diff
@@ -1895,6 +1893,18 @@ counterparts for default controllers, as discussed in [[#rs-abstract-ops-used-by
  1. Perform ! [$ReadableByteStreamControllerCallPullIfNeeded$]([=this=]).
 </div>
 
+<div algorithm>
+ <dfn abstract-op lt="[[ReleaseSteps]]" for="ReadableByteStreamController">\[[ReleaseSteps]]()</dfn>
+ implements the [$ReadableStreamController/[[ReleaseSteps]]$] contract. It performs the following
+ steps:
+
+ 1. If [=this=].[=ReadableByteStreamController/[[pendingPullIntos]]=] is not [=list/is empty|empty=],
+  1. Let |firstPendingPullInto| be [=this=].[=ReadableByteStreamController/[[pendingPullIntos]]=][0].
+  1. Set |firstPendingPullInto|'s [=pull-into descriptor/reader type=] to "`none`".
+  1. Set [=this=].[=ReadableByteStreamController/[[pendingPullIntos]]=] to the [=list=]
+     « |firstPendingPullInto| ».
```

Comment:
Alternatively, we could write:
> Remove all pull-into descriptors after *firstPendingPullInto* from this.\[[pendingPullIntos]].

or:
> Remove all pull-into descriptors whose index is not 0 from this.\[[pendingPullIntos]].

but I don't know if that's a correct usage of [remove](https://infra.spec.whatwg.org/#list-remove). 🙃

Reply:
"remove" is meant to be fairly flexible in the predicate it takes, so either of those would work. But I like what you have here; it's pretty obvious at a glance what the result will be.